assembl/static2/js/app/components/navbar/navbar.jsx

```diff
@@ -179,12 +180,18 @@ export class AssemblNavbar extends React.PureComponent {
       logoSrc: logo,
       helpUrl: helpUrl,
       location: location,
-      logoLink: sections.length > 0 && sections.find(section => section && section.sectionType === 'HOMEPAGE').url
+      logoLink: sections.length > 0 ? sections.find(section => section && section.sectionType === 'HOMEPAGE').url : ''
```

Comment:
Leaving URL as `''` might mean that `Link` tag will route to a nothing path? Or is there error handling downstream?